### DISCUSSION TITLE
New Release Tag Availability Date

### DISCUSSION THREAD

**Discussion by manish222agr** [dexidp/dex#3260] [Q&A] [answered]
Hi,

We has a release for v2.37.0 more than 6 months back. When can I expect that there will be new tag.

I am looking for new version because there are some vulnerabilities scan reports, and want to upgrade to newer version. 

Thanks.
Manish

**Comment by nabokihms** [comment, marked as answer]
This month will be a new release.

**Reply by sagikazarmark** [reply]
I can take this one if you are busy.

**Comment by nabokihms** [comment]
BTW, this year, I plan to release Dex every two months despite the number of merged PRs just to cover updates/security patches.

**Reply by sagikazarmark** [reply]
Yeah, I think that makes sense. Should we put something on the calendar @nabokihms ?

Maybe a reminder a week before the target release date, so we have time to prepare?

**Reply by nabokihms** [reply]
I've sent you an invitation 🙂

**Reply by sagikazarmark** [reply]
Thanks!
